### PR TITLE
fix-long notes on mobile devices

### PR DESCRIPTION
**Pull request by sruthiragupathy**

### CODE REVIEW

File: components/Piano/Piano.tsx [hunk 1]
```diff
@@ -96,6 +96,8 @@ const _Piano: FunctionComponent<PianoProps> = ({
             onMouseUp={() => onMouseUp(midi)}
             onMouseEnter={isMousePressed ? () => play(midi) : undefined}
             onMouseLeave={() => stop(midi)}
+            onTouchStart={()=>onMouseDown(midi)}
+            onTouchEnd={()=>onMouseUp(midi)}
```
Comment (ritz078) [inline components/Piano/Piano.tsx:100]: I guess you pushed this by disabling husky ? That's why [prettier](https://prettier.io/) didn't run automatically when you committed the code. Can you please run `yarn format` and push the changed files? That will fix the formatting here.

Comment (sruthiragupathy) [reply components/Piano/Piano.tsx:100]: Sure i will do it

**Review by ritz078 — APPROVED** [review]
Great job on this PR. 
Thank you for your contribution 🎉

**Comment by ritz078** [comment]
@all-contributors please add @sruthiragupathy for code.

**Pull request merged**